### PR TITLE
Fix issue when you pass a NULL param to the controller in Rails 5

### LINKED ISSUE #408: empty string in Rails 5 parameters will cause load_resource to raise error
I'm seeing `load_resource` raising errors when the value for the specified key is `nil` on the request.

### Example
Controller
```ruby
class UserController < ApplicationController
  load_resource :user, only: [:create, :destroy]

  def create; end
  def destroy; end
end
```
Request
```
POST :create, params: { user_id: nil }
```

In Rails 5, param values passed to the controller would all be interpreted as strings. Therefore, if `user_id` in the request above will look like `{ user_id: '' }` inside the controller. This will cause `load_resource` to raise an error because CanCan will try to do a `User.find('_id')`

https://github.com/CanCanCommunity/cancancan/blob/v1.15.0/lib/cancan/controller_resource.rb#L136-L146

### PR DESCRIPTION
**Pull request by darwin67**
resolves #408

### CODE REVIEW

File: lib/cancan/controller_resource.rb [hunk 1]
```diff
@@ -138,7 +138,7 @@ def parent_authorization_action
     end
 
     def id_param
-      @params[id_param_key].to_s if @params[id_param_key]
+      @params[id_param_key].to_s if @params[id_param_key].present?
```
Comment (coorasse) [inline lib/cancan/controller_resource.rb]: can you please add a test which fails before your fix and that is fixed now? I also change the base to `feature/v2` branch so we can release this with next major release. Thanks!

**Comment by darwin67** [comment]
@coorasse added the test for the change

**Comment by darwin67** [comment]
i'm pretty sure the test failure have nothing to do with my changes...

**Comment by darwin67** [comment]
looks like there's an invalid test configuration for Rails 5.
Rails 5 doesn't work below Ruby `2.2.2`

**Pull request merged**